extensions/amp-selector/0.1/amp-selector.js

```diff
@@ -467,11 +471,14 @@ export class AmpSelector extends AMP.BaseElement {
   selectionKeyDownHandler_(event) {
     const keyCode = event.keyCode;
     if (keyCode == KeyCodes.SPACE || keyCode == KeyCodes.ENTER) {
+      let target = event.target;
       if (this.options_.includes(event.target)) {
         event.preventDefault();
-        const el = dev().assertElement(event.target);
-        this.onOptionPicked_(el);
+      } else {
```

Comment:
I totally understand why this is reasonable for the autosuggest, but does this seem like a reasonable update to `<amp-selector>` globally? Same question re the `invocate` action registered above. 

/to @aghassemi @cvializ

Reply:
Nvm. This is past me code and also past me being dumb. Let me check this again...